app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -125,24 +133,24 @@ export class SemiCircle extends React.Component {
     100% { offset-distance : ${arcEndPercent}% }
     `;
 
+    const styleSemiCircle = () => {
+      if (nightMode) return colors.lightBlue;
+      if (cityImage === bldImage) return colors.lightYellow;
+      return colors.lightPink;
+    };
+
     return (
       <svg width={width} height={height}>
         <defs>
           <radialGradient id="circleGradient">
-            <stop
-              offset="5%"
-              stopColor={nightMode ? colors.lightBlue : colors.lightPink}
-            />
+            <stop offset="5%" stopColor={styleSemiCircle()} />
             <stop offset="95%" stopColor={colors.black} />
           </radialGradient>
         </defs>
         <defs>
           <linearGradient id="strokeGradient">
             <stop offset="2%" stopColor={colors.black} />
-            <stop
-              offset="50%"
-              stopColor={nightMode ? colors.blue : colors.pink}
-            />
+            <stop offset="50%" stopColor={styleSemiCircle()} />
```

Comment:
There's a subtle bug here with the color refactor. Maybe its worthwhile to define our day/night colors per component?

Reply:
This should be fixed now!